fontbe/src/os2.rs

```diff
@@ -0,0 +1,27 @@
+//! Generates a [OS/2](https://learn.microsoft.com/en-us/typography/opentype/spec/os2) table.
+
+use fontdrasil::orchestration::Work;
+use write_fonts::tables::os2::Os2;
+
+use crate::{
+    error::Error,
+    orchestration::{BeWork, Context},
+};
+
+struct Os2Work {}
+
+pub fn create_os2_work() -> Box<BeWork> {
+    Box::new(Os2Work {})
+}
+
+impl Work<Context, Error> for Os2Work {
+    /// Generate [OS/2](https://learn.microsoft.com/en-us/typography/opentype/spec/os2)
+    fn exec(&self, context: &Context) -> Result<(), Error> {
+        //let static_metadata = context.ir.get_final_static_metadata();
+        let os2 = Os2 {
+            ..Default::default()
```

Comment:
curious what does that contain? is it literally all zeros?

Reply:
```
  <OS_2>
    <!-- The fields 'usFirstCharIndex' and 'usLastCharIndex'
         will be recalculated by the compiler -->
    <version value="0"/>
    <xAvgCharWidth value="0"/>
    <usWeightClass value="400"/>
    <usWidthClass value="5"/>
    <fsType value="00000000 00000000"/>
    <ySubscriptXSize value="0"/>
    <ySubscriptYSize value="0"/>
    <ySubscriptXOffset value="0"/>
    <ySubscriptYOffset value="0"/>
    <ySuperscriptXSize value="0"/>
    <ySuperscriptYSize value="0"/>
    <ySuperscriptXOffset value="0"/>
    <ySuperscriptYOffset value="0"/>
    <yStrikeoutSize value="0"/>
    <yStrikeoutPosition value="0"/>
    <sFamilyClass value="0"/>
    <panose>
      <bFamilyType value="0"/>
      <bSerifStyle value="0"/>
      <bWeight value="0"/>
      <bProportion value="0"/>
      <bContrast value="0"/>
      <bStrokeVariation value="0"/>
      <bArmStyle value="0"/>
      <bLetterForm value="0"/>
      <bMidline value="0"/>
      <bXHeight value="0"/>
    </panose>
    <ulUnicodeRange1 value="00000000 00000000 00000000 00000000"/>
    <ulUnicodeRange2 value="00000000 00000000 00000000 00000000"/>
    <ulUnicodeRange3 value="00000000 00000000 00000000 00000000"/>
    <ulUnicodeRange4 value="00000000 00000000 00000000 00000000"/>
    <achVendID value="    "/>
    <fsSelection value="00000000 00000000"/>
    <usFirstCharIndex value="0"/>
    <usLastCharIndex value="0"/>
    <sTypoAscender value="0"/>
    <sTypoDescender value="0"/>
    <sTypoLineGap value="0"/>
    <usWinAscent value="0"/>
    <usWinDescent value="0"/>
  </OS_2>
```